thewalrus/tests/test_montrealer.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain adj copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Montrealer tests"""
+
+import pytest
+import numpy as np
+from thewalrus import mtl, lmtl
+from thewalrus.reference import rspm, rpmp, mtl as mtl_symb, lmtl as lmtl_symb
+from scipy.special import factorial2
+from scipy.stats import unitary_group
+
+
+@pytest.mark.parametrize("n", range(1, 6))
+def test_montrealer_all_ones(n):
+    """Test that the Montrealer of a matrix of ones gives (2n-2)!!"""
+    adj = np.ones([2 * n, 2 * n])
+    mtl_val = mtl(adj)
+    mtl_expect = factorial2(2 * n - 2)
+    assert np.allclose(mtl_val, mtl_expect)
+
+
+@pytest.mark.parametrize("n", range(1, 8))
+def test_size_of_pmpr(n):
+    """rpmp(2n) should have (2n-2)!! elements"""
```

Comment:
the name of the function and the docstring do not match

Reply:
@yaniccd how about this one?

Reply:
Fixed.